docs/source/transaction_family_specifications/track_and_trade_transaction_family.rst

```diff
@@ -0,0 +1,717 @@
+older updates are overwritten.
+
+Updates to Properties can have one of the following protobuf types:
+`bytes`, `string`, `sint64`, `float`, or `Location` (see the section
+on RecordTypes below). The type of an update is indicated by a tag
+belonging to the PropertySchema object.
+
+.. code-block:: protobuf
+
+   message Property {
+       // The name of the Property, e.g. `temperature`.
+       string name = 1;
+
+       // The identifier of the Property's associated Record.
+       string record_id = 2;
+       TODO - reporter index, etc
+
+       // The Property's type (int, string, etc.)
+       PropertySchema.DataType data_type = 3;
+
+       // The public keys of reporters authorized to send updates.
+       // To maintain an accurate index, new reporters are appended,
+       // reporters whose authorization has been revoked are not
+       // deleted, but rather set to null.
+       repeated string reporters = 3;
+
+       // The page to which new updates are added. This number represents
+       // the last 4 hex characters of the page's address. Consequently,
+       // it should not exceed 16^4 = 65536.
+       uint32 current_page = 4;
+
+       // A flag indicating whether the first 16^4 pages have been filled.
+       // This is used to calculate the last four hex characters of the
+       // address of the page containing the earliest updates. When it is
+       // false, the earliest page's address will end in `0001`. When it is
+       // true, the earliest page's address will be one more than the
+       // current_page, or `0001` if the current_page is `ffff`.
+       bool wrapped = 5;
+   }
+
+   message PropertyPage {
+       message ReportedValue {
+           // The index of the reporter id in reporters field
+           uint32 reporter_index = 1;
+           // Approximately when this value was reported, as a Unix UTC timestamp
+           uint64 timestamp = 2;
+
+           // The type-specific value of the update. Only one of these
+           // fields should be used, and it should match the type
+           // specified for this Property in the RecordType.
+           bytes bytes_value = 11;
+           string string_value = 12;
+           sint64 int_value = 13;
+           float float_value = 14;
+           Location location_value = 15;
+       }
+
+       string name = 1;
+       string record_id = 2;
+       PropertySchema.DataType data_type = 3;
+
+       // ReportedValues are sorted first by timestamp, then by reporter_index
+       repeated ReportedValue reported_values = 4;
+   }
+
+
+Properties and PropertyPages whose addresses collide will be stored in
+lists alphabetized by Property name.
+
+.. code-block:: protobuf
+
+   message PropertyContainer {
+       repeated Property entries = 1;
+   }
+
+   message PropertyPageContainer {
+       repeated PropertyPage entries = 1;
+   }
+
+
+Record Types
+------------
+
+To simplify how tracking data is handled for each Record, Records will
```

Comment:
Do RecordTypes "simplify" how tracking data is handled? This feels both nonspecific and inaccurate. Perhaps: _"In order to validate incoming tracking data, Records will . . ."_.